app/controllers/shared_secret_authenticated_api_controller.rb

```diff
@@ -16,6 +16,6 @@ def authenticate
   end
 
   def shared_secret
-    ENV[SHARED_SECRET_ENV_VAR]
+    ENV.fetch(SHARED_SECRET_ENV_VAR, nil)
```

Comment:
Just confirming, the `nil` return is fairly safe here because up in `authenticate` we're `raise`ing if it's nil.